english/sc2/sc2.md

```diff
@@ -0,0 +1 @@
+It's 10 p.m. and you're borging, and decide to surf on google, for stacraft 1, an old game that you play some years ago, suddenly you read starcraft 2 a new version with beter graphics and you give it a chance again visiting [Starcraft 2](www.starcraft2.com)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.